docs/server/source/simple-network-setup.md

```diff
@@ -0,0 +1,197 @@
+
+```bash
+sudo pip3 install bigchaindb==2.0.0a5
+```
+
+Check you installed the correct version of BigchainDB Server with `bigchaindb --version`.
+
+#### Install MongoDB
+Install a recent version of MongoDB. BigchainDB Server requires `3.4` or newer.
+
+```bash
+sudo apt install mongodb
+```
+
+The package manager should take care of installing the startup script for MongoDB as well.
+
+#### Install Tendermint
+Install a [recent version of Tendermint][tendermint:releases]. BigchainDB Server requires `0.19` or newer.
+
+```bash
+sudo apt install unzip
+wget https://github.com/tendermint/tendermint/releases/download/v0.19.3/tendermint_0.19.3_linux_amd64.zip
+unzip tendermint_0.19.3_linux_amd64.zip
+rm tendermint_0.19.3_linux_amd64.zip
+sudo mv tendermint /usr/local/bin
+```
+
+### Firewall settings
+Make sure to accept inbound connections to (`22`, you don't want to lock you out), `9984`, `9985`, and `46656`.
+
+```bash
+sudo ufw allow 22/tcp
+sudo ufw allow 9984/tcp
+sudo ufw allow 9985/tcp
+sudo ufw allow 46656/tcp
+sudo ufw enable
+```
+
+Some cloud providers, like Microsoft Azure, require you to go through their portal to allow inbound connections on custom ports.
+
+## Member: Configure BigchainDB Server
+To configure BigchainDB Server, run:
+
+```bash
+bigchaindb configure
+```
+
+The first question is ``API Server bind? (default `localhost:9984`)``, to expose the API to the public, bind the API Server to `0.0.0.0:9984`. Unless you have specific needs, you can keep the default value for all other questions.
+
+## Member: Generate the private key and address
+A Node is identified by the triplet `<hostname, address, public_key>`.
+
+As a Member, it's your duty to create and store securely your private key, and share your `hostname`, `address`, and `public_key` with the other members of the network.
+
+To generate all of that, run:
+
+```bash
+tendermint init
+```
+
+The `public_key` and `address` are stored in the file `.tendermint/config/priv_validator.json`, and it should look like:
+
+```json
+{
+  "address": "5943A9EF6285791A504918E1D117BC7F6A615C98",
+  "pub_key": {
+    "type": "AC26791624DE60",
+    "value": "W3tqeMCU3d4SHDKqrwQWTahTW/wpieIAKZQxUxLv6rI="
+  },
+  "last_height": 0,
+  "last_round": 0,
+  "last_step": 0,
+  "priv_key": {
+    "type": "954568A3288910",
+    "value": "3sv9aExgME6MMjx0JoKVy7KtED8PBiPcyAgsYmVizslbe2p4wJTd3hIcMqqvBBZNqFNb/CmJ4gAplDFTEu/qsg=="
+  }
+}
+```
+
+Share with all other Members your `address`, `pub_key.value`, and the host name or IP of your Node.
+
+**Important note on security: each Member should verify the data they receive from the other Members.**
+
+## Coordinator: Initialize the Network
+At this point the Coordinator should has received the data from all the Members, and should combine them in the `.tendermint/config/genesis.json` file:
+
+```json
+{
+  "genesis_time": "0001-01-01T00:00:00Z",
+  "chain_id": "test-chain-la6HSr",
```

Comment:
It's important that everyone shares the same `chain_id`.

Reply:
True, I'll make it more explicit thanks!